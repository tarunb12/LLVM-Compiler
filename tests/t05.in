int main() {
    printf(2 + 3 = 4 + 1)
}