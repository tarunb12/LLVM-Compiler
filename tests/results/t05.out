Error: File "tests/t05.in": Invalid first argument for printf: 2 + 3 = 4 + 1.